README.md

```diff
@@ -172,58 +207,189 @@ if (group != null && user != null)
 }
 ```
 
-### Retrieving a User's Groups
+### List a User's enrolled Factors
+
 ``` csharp
-// find the desired user
-var user = await _oktaClient.Users.FirstOrDefault(x => x.Profile.Email == "laura.rodriguez@okta.com");
+// Find the desired user
+var user = await client.Users.FirstOrDefault(x => x.Profile.Email == "darth.vader@imperial-senate.gov");
 
-// get the user's groups
-var groups = await user.Groups.ToList();
+// Get user factors
+var factors = await user.Factors.ToArray();
+```
+
+### Enroll a User in a new Factor
+``` csharp
+// Find the desired user
+var user = await client.Users.FirstOrDefault(x => x.Profile.Email == "darth.vader@imperial-senate.gov");
+
+// Enroll in Okta SMS factor
+await user.AddFactorAsync(new AddSmsFactorOptions
+                {
+                    PhoneNumber = "+99999999999",
+                });
 ```
+### Activate a Factor
+``` csharp
+// Find the desired user
+var user = await client.Users.FirstOrDefault(x => x.Profile.Email == "darth.vader@imperial-senate.gov");
 
-## Using raw HTTP methods
+// Find the desired factor
+var smsFactor = await user.Factors.FirstOrDefault(x => x.FactorType == FactorType.Sms);
 
-The SDK client object can be used to make calls to any Okta API (not just the endpoints officially supported by the SDK) via the `GetAsync`, `PostAsync`, `PutAsync` and `DeleteAsync` methods. You can take a look at [GitHub](https://github.com/okta/okta-sdk-dotnet/blob/master/src/Okta.Sdk/OktaClient.cs) to see the different overloadings.
+// Activate sms facotr
+await client.UserFactors.ActivateFactorAsync(verifyFactorRequest, user.Id, smsFactor.Id);
+```
 
-### Activating a User via `PostAsync`
+### Verify a Factor
+``` csharp
+// Find the desired user
+var user = await client.Users.FirstOrDefault(x => x.Profile.Email == "darth.vader@imperial-senate.gov");
 
-```csharp
-var userId = "<Some User ID String or Login>";
+// Find the desired factor
+var smsFactor = await user.Factors.FirstOrDefault(x => x.FactorType == FactorType.Sms);
 
-await _oktaClient.PostAsync(
-    new Okta.Sdk.Internal.HttpRequest
+// Verify sms factor
+var response = await client.UserFactors.VerifyFactorAsync(verifyFactorRequest, user.Id, smsFactor.Id);
+```
+
+### List all Applications
+``` csharp
+// List all applications
+var appList = await client.Applications.ListApplications().ToArray();
```

Comment:
`await client.Applications.ToArray()` should work here right? I prefer the simpler syntax.

Reply:
`ApplicationsClient` implements ` IAsyncEnumerable<IApplication>` but I forgot to add this in the `IApplicationsClient`.  I will push this change before publishing on Nuget, will tell Rob to review.